wraith/wraith.md

```diff
@@ -171,24 +126,34 @@ for [pet '("cat" "dog" "horse")]          |  (for [(pet '("cat" "dog" "horse"))]
 define (counting-letters-song letters)    |  (define (counting-letters-song letters)
   for [letter letters                     |    (for [(letter letters)
        number (in-naturals 1)]            |          (number (in-naturals 1))]
-    printf "I like ~a, it's number ~a!" \ |      (printf "I like ~a, it's number ~a!"
-      letter number                       |         letter number)
-    (newline)                             |      (newline))
+    printf "I like ~a, it's number ~a!"   |      (printf "I like ~a, it's number ~a!"
+      . letter number                     |         letter number)
+    newline                               |      (newline))
   displayln "Singing a letters song!"     |    (displayln "Singing a letters song!"))
 ```
 
+The first line of the bracketized expression is always closed at the
+end of the line. To use a multi-line expression, it must be on its own line.
+
 ``` racket
-let* [animal "dog"                        |  (let* [(animal "dog")
-      noise "barks"                       |         (noise "barks")
-      player-hears                        |         (player-hears
-        format "the ~a says: ~a!!!" \     |          (format "the ~a says: ~a!!!"
-               animal noise]              |                  animal noise))]
-  displayln player-hears                  |    (displayln player-hears))
+define (greeter name)                     |  (define (greeter name)
+  let ((to-say                            |    (let ((to-say
+          format "Hey there ~a! :D"       |           (format "Hey there ~a! :D"
+                 . name))                 |                   name)))
+    displayln to-say                      |      (displayln to-say)))
 ```
 
-However, in Wraith, new lines of content within parenthetical/bracketed
-expressions are not permitted to have less indentation than the column
-after the opening parenthesis.
+``` racket
+define (greeter name)                     |  (define (greeter name)
+  let [                                   |    (let (
+       to-say                             |          (to-say
+          format "Hey there ~a! :D"       |           (format "Hey there ~a! :D"
+                 . name]                  |                   name)))
+    displayln to-say                      |      (displayln to-say)))
+```
+
```

Comment:
this special-case could be avoided with the more complex parser described in the pull-request and comment-description.